fuzzers/libfuzzer_dataflow_pre/builder.Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
```

Comment:
Instead of adding this, can we just update the current version of libFuzzer?

Reply:
Sure! That would be much easier.
I add this only because I don't want to let my experiment affect other experiments that might rely on the specific version of `LibFuzzer`.